packages/base-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.3]
+
+### Changed
+
+- Refactor TokensController tests ([#4247](https://github.com/MetaMask/core/pull/4247))
```

Comment:
This change was just to unit tests, it doesn't affect users of this package in any way. We can omit a version bump for this package, and remove this from the changelog

Reply:
Oh never mind, this PR does actually affect the NetworkController in a tangible way. See the PR description for a more accurate change entry there

Reply:
Mark,

Thank you for taking a closer look. I had seen the note about removing the entries that "don't affect consumers". When I removed those entries, however, the changelog for version 5.0.3 was empty. I hesitated to remove the entire version bump from the changelog, as the version is also in package.json.

I've gone ahead and reverted the basecontroller/CHANGELOG as well as basecontroller/package.json. 

For the material change to NetworkController I think you are referring to the [newly-exported members](https://github.com/MetaMask/core/pull/4247/files#diff-ea0aa37793484e278bd59970b7a6ce8380b75b723b34612081d402fb66261cfd).

Reply:
Yeah that was the one! Thanks